packages/docregistry/src/default.ts

```diff
@@ -227,12 +227,15 @@ export class TextModelFactory implements DocumentRegistry.CodeModelFactory {
    * Create a new model.
    *
    * @param languagePreference - An optional kernel language preference.
+   * @param modelDB - An optional modelDB.
+   * @param isInitialized - An optional flag to check if the model is initialized.
    *
    * @returns A new document model.
    */
   createNew(
     languagePreference?: string,
-    modelDB?: IModelDB
+    modelDB?: IModelDB,
+    isInitialized?: boolean
```

Comment:
Note to self: check whether this new parameter should be considered a breaking change.

Reply:
It is tagged as optional with `?`, so it should be backwards compatible.